common/lib/xmodule/xmodule/modulestore/tests/test_mixed_modulestore.py

```diff
@@ -257,7 +240,7 @@ def _has_changes(self, location):
         return self.store.has_changes(self.store.get_item(location))
 
     # pylint: disable=dangerous-default-value
-    def _initialize_mixed(self, mappings=MAPPINGS, contentstore=None):
+    def _initialize_mixed(self, mappings={}, contentstore=None):
```

Comment:
I realize where it comes from, but could you please make mappings default to `None` and then do a `mappings = mappings or {}` below? Mutable default args are scary unless there's a really good, commented reason for it.